javascript/javascript2/week1/scripts.js

```diff
@@ -0,0 +1,58 @@
+//Shortest word
+console.log("TASK: Shortest Word")
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+shortestWord(danishWords); // returns 'ø'
+
+function shortestWord(list){ 
+    let shortestWord = list[0].length;
+    for(i=0;i<list.length;i++){
+        if(shortestWord > list[i].length){
+            shortestWord = i;
+        }
+    }
```

Comment:
good solution no need to change it.
check out a cooler one though using the reduce function : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce
```
listreduce((a, b) => a.length <= b.length ? a : b)
```